### PR TITLE
Fixes after Design Review

### PR DESCRIPTION
**Pull request by davidfloegel**

### CODE REVIEW

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/131
Branch:
https://sofarsounds.github.io/maestro/branches/design-review-fixes
Storybook:
https://sofarsounds.github.io/maestro/5024f8cbe9f7ac6e9b0eada65b8c4f663d85cf45/
Build:
https://sofarsounds.github.io/maestro/5024f8cbe9f7ac6e9b0eada65b8c4f663d85cf45/dist.zip

File: src/molecules/Card/CardContainer.tsx [hunk 1]
```diff
@@ -20,6 +21,16 @@ export default styled.div<CardContainerProps>`
       ${Title}, ${Preview}, ${Footer} {
         color: ${theme.colours.whiteDenim};
       }
+    `}
+
+    ${clickable &&
+      css`
+        &:hover {
+          ${Title} {
```
Comment (liondedan) [inline src/molecules/Card/CardContainer.tsx]: 👍

**Comment by davidfloegel** [comment]
# GitHub Pages Deployment
Pull Request:
https://sofarsounds.github.io/maestro/pull/131
Branch:
https://sofarsounds.github.io/maestro/branches/design-review-fixes
Storybook:
https://sofarsounds.github.io/maestro/3ccdfeb1e76560fd1788ad388ca92d3e678d3809/
Build:
https://sofarsounds.github.io/maestro/3ccdfeb1e76560fd1788ad388ca92d3e678d3809/dist.zip

**Comment by davidfloegel** [comment]
@liondedan I did typing exercises on a weighted keyboard on holiday so now I'm typing 400 chars per minute.

**Pull request merged**